.github/workflows/main.yml

```diff
@@ -680,5 +680,5 @@ jobs:
         run: |
           go env -w GOFLAGS=-mod=mod
           go install github.com/google/addlicense@2fe3ee94479d08be985a84861de4e6b06a1c7208
-          addlicense -ignore "**/script-empty.sh" -ignore "pkg/testdata/**" -ignore "checks/testdata/**" -l apache -c 'Security Scorecard Authors' -v *
+          addlicense -s -ignore "**/script-empty.sh" -ignore "pkg/testdata/**" -ignore "checks/testdata/**" -l apache -c 'OpenSSF Authors' -v *
```

Comment:
I thought we decided this should continue to be `Scorecard Authors` but without the year/date? @justaugustus

Reply:
> I thought we decided this should continue to be `Scorecard Authors` but without the year/date? @justaugustus

From https://github.com/ossf/scorecard/issues/1534#issuecomment-1048102540:

> So I guess to more concretely answer this, a few things need to happen:
> 
>     1. Scorecard maintainers (@naveensrinivasan @azeemshaikh38 @laurentsimon me) need to decide if it's fine to elide the copyright date in all files (I'm personally fine with it)
> 
>     2. OpenSSF and scorecard maintainers need to decide if it's preferred to use `OpenSSF Authors` as the copyright holder for content in this repo and potentially elsewhere
> 
>     3. Someone (@singhsaurabh) will update the copyright headers in this repo (as well as any tests that enforce them) based on the decisions on 1 and 2

I thought we landed on 2: `OpenSSF Authors` (no date) or rather, haven't heard anything to the contrary.

Reply:
> I thought we landed on 2: `OpenSSF Authors` (no date) or rather, haven't heard anything to the contrary.

In the last Scorecard bi-weekly we decided on `1` (see https://github.com/ossf/scorecard/discussions/1684)? `2` would be ideal, but we need confirmation from LF for this.